### PR TITLE
Fix item.name in the enroll-cluster tasks and execute import-cluster script

### PR DESCRIPTION
**Pull request by serngawy**

### CODE REVIEW

File: acm-spoke/inventory/hosts.sample [hunk 1]
```diff
@@ -1,6 +1,7 @@
 [all:vars]
 
 kubeconfig_hub_path=/path/to/acm_hub_kubeconfig
+out_dir=/tmp
```
Comment (yrobla) [inline acm-spoke/inventory/hosts.sample:4]: can you reuse the temporary_path entry, that already exists on the inventory?

**Review by yrobla — CHANGES_REQUESTED** [review]

File: acm-spoke/inventory/hosts.sample (hunk 1 above)
Comment (serngawy) [reply acm-spoke/inventory/hosts.sample:4]: there is no temporary_path entry already exist

File: acm-spoke/enroll-clusters.yaml [hunk 2]
```diff
@@ -7,15 +7,15 @@
   shell: "for kf in $(find {{ acm_directory.path }}/subscriptions/ -name kustomization.yaml); do oc --kubeconfig={{ kubeconfig_hub_path }} apply -k $(echo $kf | rev | cut -d'/' -f 2- | rev); done"
 
 - name: Import cluster to ACM
-  shell: "import_cluster.sh {{ hostvars[item].cluster_name }} {{ kubeconfig_hub_path }} {{ hostvars[item].profile }}"
+  shell: "{{ temporary_path }}/import_cluster.sh {{ hostvars[item].name }} {{ hostvars[item].profile }} {{ kubeconfig_hub_path }} {{ temporary_path }}"
 
 - name: Extract the klusterlet definition from ACM
-  shell: "oc get secret/{{ hostvars[item].cluster_name }}-import -n {{ hostvars[item].cluster_name }} -o jsonpath={.data.crds\\\\.yaml} | base64 --decode > {{ acm_directory.path }}/crd.yaml"
+  shell: "oc get secret/{{ hostvars[item].name }}-cluster-import -n {{ hostvars[item].name }} -o jsonpath={.data.crds\\\\.yaml} | base64 --decode > {{ acm_directory.path }}/crd.yaml"
```
Comment (yrobla) [inline acm-spoke/enroll-clusters.yaml:13]: why are you adding the -cluster here? shouldn't that just be the name of the cluster as before?

Comment (serngawy) [reply acm-spoke/enroll-clusters.yaml:13]: Actually it wasn't working, the name of the cluster in ran manifest having "cluster"
https://github.com/redhat-ztp/ztp-ran-manifests/blob/main/spoke_clusters/01_sample_cu_cluster.yaml#L14

**Review by yrobla — APPROVED** [review]

**Pull request closed without merging**